lib/api/navigation.js

```diff
@@ -0,0 +1,130 @@
+var linz = require('../../'),
+    utils = require('../utils'),
+    async = require('async');
+
+function get (req, cb) {
+
+    if (!linz.get('disable navigation cache') && !req.linz.cache.navigation.invalidate && req.session.navigation) {
+        return cb(null, req.session.navigation);
+    }
+
```

Comment:
Upon running with this new build, I received the message 5 times. I think it should only be once?